sample-apps/delayed_job-sample/Procfile

```diff
@@ -1,4 +1,4 @@
 release: bundle exec rails db:migrate
 proxy: npx judoscale-adapter-proxy-server
-rails: rails server
-dj: DYNO=worker.1 QUEUES=default,low,high bundle exec rake jobs:work
+rails: bundle exec rails server
+dj: DYNO=worker.1 RENDER_SERVICE_ID=srv-abcdefg RENDER_INSTANCE_ID=srv-abcdefg-1234-1234 RENDER_SERVICE_TYPE=worker QUEUES=default,low,high bundle exec rake jobs:work
```

Comment:
@jon-sully Thoughts on this?

Reply:
I feel like there was a particular reason I set them separately and in both, but it's eluding me at the moment 🤔 I'll spin back up on this tomorrow and see if I can remember, otherwise condense 👍

Reply:
Circled back on this one this morning and realized that I was being specific with ENV vars in both layers in order to get the env vars correct for each process depending on whether we wanted to spoof Heroku or Render locally (which.. is probably of little value to begin with) but decided to scrap all that and go a different, more explicit way. Now the "switch between Heroku and Render" logic is in the Procfiles themselves and each process will get their own ENV vars coded in there:

https://github.com/judoscale/judoscale-ruby/pull/142/files#diff-3bcfd44816e08c13638a2aee5287aa21b432eae56171b86572142c3f380de40f

```procfile
proxy: npx judoscale-adapter-proxy-server

# NOTE: Enable either the `heroku_` processes or the `render_` processes based
# on which env you want to spoof locally.

heroku_rails: DYNO=web.1 bundle exec rails server
heroku_resque: DYNO=worker.1 COUNT=2 QUEUE=* bundle exec rake resque:workers

# render_rails: RENDER_SERVICE_ID=srv-xyz RENDER_INSTANCE_ID=srv-xyz-1234 RENDER_SERVICE_TYPE=web bundle exec rails server
# render_resque: RENDER_SERVICE_ID=srv-abc RENDER_INSTANCE_ID=srv-abc-1234 RENDER_SERVICE_TYPE=worker COUNT=2 QUEUE=* bundle exec rake resque:workers
```